lib/fluent/plugin/in_kafka.rb

```diff
@@ -114,8 +116,10 @@ def configure(conf)
     @parser_proc = setup_parser
 
     @time_source = :record if @use_record_time
+    @time_key = 'time'
```

Comment:
Why separate instance variable into `@time_key` and `@record_time_key`?
Does using `@record_time_key` for `record[]` have a problem?

Reply:
You have a good point, I will update it.